runtime/runc_jailer.go

```diff
@@ -353,19 +379,11 @@ func (j runcJailer) jailerCommand(containerName string, isDebug bool) *exec.Cmd
 }
 
 // overwriteConfig will set the proper default values if a field had not been set.
-//
-// TODO: Add netns
-func (j runcJailer) overwriteConfig(cfg *Config, socketPath, configPath string) error {
-	spec := specs.Spec{}
-	configBytes, err := ioutil.ReadFile(configPath)
-	if err != nil {
-		return err
-	}
-
-	if err := json.Unmarshal(configBytes, &spec); err != nil {
-		return err
-	}
-
+func (j *runcJailer) overwriteConfig(cfg *Config, machineConfig *firecracker.Config, socketPath, configPath string) error {
+	var err error
+	// here we attempt to cache the runc config. If the config has already been
+	// cached, we will return immediately
```

Comment:
Is this comment still relevant? Otherwise LGTM

Reply:
Good catch, fixed!